app/models/user/tasks/incomplete_agreement.rb

```diff
@@ -0,0 +1,44 @@
+module User
+  module Tasks
+    class IncompleteAgreement
+      include ActiveModel::Model
```

Comment:
@developish - I see you're using ActiveModel::Model now. Do you recommend we continue to use this for anything related to the new CurrentUser model?

Reply:
Not necessarily, but I think this is better than the big bag of `data` that I used before. We can discuss when and why to use it.